airgap/terraform/variables.tf

```diff
@@ -53,3 +53,13 @@ variable "aws_ssh_private_key_file_path" {
   type        = string
   default     = "~/.ssh/id_rsa"
 }
+
+variable "appco_test" {
```

Comment:
The variable doesn't have a default value, which will cause the existing air-gap pipeline to fail: https://ci.longhorn.io/job/private/job/longhorn-helm-chart-test/57/

Reply:
Updated